.goreleaser.yaml

```diff
@@ -12,7 +12,7 @@ env:
   - CC_darwin_arm64=oa64-clang
   - CXX_darwin_arm64=oa64-clang++
   - CC_darwin_amd64=o64-clang
-  - CXX_darwin_amd64=o64-clang+
+  - CXX_darwin_amd64=o64-clang++
```

Comment:
was this not working before? wondering how we didnt spot something failing, nice catch

do we still need this file? i see there is an issue to remove legacy makefile cmds, i guess this one will be removed then as well